docs/guides/tools/truffle.md

```diff
@@ -1,49 +1,51 @@
 <!--
-order: 1
+order: 4
 -->
 
-# Truffle
+# Truffle: Deploying a Smart Contract
 
-Set up a Truffle Ethermint local development environment. {synopsis}
+Learn how to deploy a simple Solidity-based smart contract to Ethermint using the Truffle environment {synopsis}
 
 ## Pre-requisite Readings
 
 - [Installation](./../quickstart/installation.md) {prereq}
 - [Run a node](./../quickstart/run_node.md) {prereq}
 
-[Truffle](https://www.trufflesuite.com/truffle) is a development framework for deploying and managing [Solidity](https://github.com/ethereum/solidity) smart contracts. In this guide, we will learn how to deploy a contract to a running Ethermint network.
+[Truffle](https://www.trufflesuite.com/truffle) is a development framework for deploying and managing [Solidity](https://github.com/ethereum/solidity) smart contracts.
 
-## Install dependencies
+## Install Dependencies
 
 First, install the latest Truffle version on your machine globally.
 
-```bash
-yarn install truffle -g
+```shell
```

Comment:
curious about the use of bash vs shell? should we enforce one over the other?

Reply:
Actually, after doing some research the answer is no. The lexers Github-flavored md uses are different than vuepress it seems.

`shell` is just an alias of `bash ` for vuepress, so using `bash` is fine. Full list [here](https://github.com/PrismJS/prism/blob/master/components.json).